src/Microsoft.AspNet.Razor/Properties/AssemblyInfo.cs

```diff
@@ -5,5 +5,4 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("Microsoft.AspNet.Razor.Test")]
-[assembly: InternalsVisibleTo("Microsoft.AspNet.Razor.Runtime.Test")]
```

Comment:
1. @NTaylorMullen's comment just above (about using `aspnet/Common` packages) is not correct. `Microsoft.AspNet.Razor.Runtime` and `Microsoft.AspNet.Razor` are already using the same `aspnet/Common` packages e.g. `Microsoft.Framework.NotNullAttribute.Sources`. Problems online arise if a test project references both assemblies (i.e. must be the `Microsoft.AspNet.Razor.Runtime.Test` project) _and_ the test project itself uses something from the duplicate `aspnet/Common` packages. Remote case and not a problem we need to fix today.
2. Because this part of the change represents churn for churn's sake.
3. We don't make changes based on personal preferences which seems to be the real issue here.